tgstation.dme

```diff
@@ -3061,8 +3061,8 @@
 #include "code\modules\reagents\chemistry\reagents\reaction_agents_reagents.dm"
 #include "code\modules\reagents\chemistry\reagents\toxin_reagents.dm"
 #include "code\modules\reagents\chemistry\reagents\impure_reagents\impure_medicine_reagents.dm"
-#include "code\modules\reagents\chemistry\reagents\unique\eigenstasium.dm"
 #include "code\modules\reagents\chemistry\reagents\impure_reagents\impure_toxin_reagents.dm"
+#include "code\modules\reagents\chemistry\reagents\unique\eigenstasium.dm"
```

Comment:
Is this change supposed to be here? This looks like what's on master already

Reply:
fixed, thanks